src/protocol/generator.py

```diff
@@ -52,7 +49,7 @@ def get_c_size(self, t):
         # this regex will get the X in u/intX_t (uint8_t, int16_t)
         match = re.search('[0-9]{1,2}', t)
         if match:
-            return int(match.group(0)) / 8
+            return int(int(match.group(0)) / 8)
```

Comment:
why is this necessary ?

Reply:
Previously, only the dividend was converted, and the return was still float. My fix here was not the best (just needed to move a parenthesis). I'll take care of it.

Reply:
(We were getting things like `msgData[2.0]`, this patch fixes that)